scala/scalafmt/BUILD

```diff
@@ -40,8 +40,11 @@ declare_deps_provider(
     visibility = ["//visibility:public"],
     deps = [
         "@com_geirsson_metaconfig_core",
+        "@org_scalameta_common",
         "@org_scalameta_parsers",
         "@org_scalameta_scalafmt_core",
+        "@org_scalameta_scalameta",
+        "@org_scalameta_trees",
```

Comment:
NB: These are already transitive dependencies of `scalafmt_core`, but with the new phase position, the test requiring explicitly-specified dependencies requires them to be listed explicitly.